test-data/genops-set.test

```diff
@@ -4,28 +4,26 @@ def f() -> Set[int]:
     return {1, 2, 3}
 [out]
 def f():
-    r0 :: set
-    r1 :: short_int
-    r2 :: object
-    r3 :: bool
-    r4 :: short_int
-    r5 :: object
-    r6 :: bool
-    r7 :: short_int
+    r0, r1, r2 :: short_int
+    r3 :: set
+    r4 :: object
+    r5 :: bool
+    r6 :: object
+    r7 :: bool
     r8 :: object
     r9 :: bool
 L0:
-    r0 = set
-    r1 = 1
-    r2 = box(short_int, r1)
-    r3 = r0.add(r2) :: set
-    r4 = 2
-    r5 = box(short_int, r4)
-    r6 = r0.add(r5) :: set
-    r7 = 3
-    r8 = box(short_int, r7)
-    r9 = r0.add(r8) :: set
-    return r0
+    r0 = 1
+    r1 = 2
+    r2 = 3
+    r3 = set
+    r4 = box(short_int, r0)
```

Comment:
It occurs to me that there is a compatibility question here, since it is observable how we order evaluating the expressions and inserting them into the set (for example, because the __eq__ or __hash__ methods could fail or have side effects). But also I am pretty sure I don't care at all about this compatibility question.